gm4_shiny_items/data/gm4_shiny_items/functions/tick.mcfunction

```diff
@@ -0,0 +1,4 @@
+
+schedule function gm4_shiny_items:tick 1t
+
+execute as @e[scores={gm4_absorbed_xp=0..}] at @s run function gm4_shiny_items:check_for_stop
```

Comment:
This is very bad for performance. Generally speaking  external 20Hz functions can be replaced by internal schedule loops nowadays.
If this needs to stay this way, trigger the execute using a fake player and add more selector arguments to the @e selector.